### ISSUE TITLE
Adding a preset for paddling pools

### ISSUE THREAD

**Issue by Vuizur** [openstreetmap/id-tagging-schema#1322]
I had an area that I wanted to mark as a paddling pool in the online id editor, however there currently is no suggested preset for this. It would be awkward to tag it as a swimming pool, because you can't swim there, so I was quite unsure. I saw that there is a "Kids pool" string in transifex, but I'm not sure for what it is used.

Link to wiki: https://wiki.openstreetmap.org/wiki/Tag:leisure%3Dpaddling_pool <- Here you can see that there are several ways to tag it, so I am not sure what the best one would be.

**Comment by tordans** [comment]
Interesting, I think it makes sense to have a preset for this.
- or a field on the swimming_pool preset to specify the usage.
- or maybe both, so user can more easily discover the sub presets when only the main preset is selected.

(Side note: I was amazed to see we don't have any of the sub presets selected [in this case in Neukölln](https://www.openstreetmap.org/way/104550137/history), which is mapped [in high detail](https://strassenraumkarte.osm-berlin.org/?map=micromap#18/52.47985/13.41608) :)).

---

Some research:

There are not wiki pages for the "possible synonyms"
- https://taginfo.openstreetmap.org/tags/swimming_pool%3Dchildren%27s_pool#wiki
- https://taginfo.openstreetmap.org/tags/swimming_pool%3Dwading#wiki
- https://taginfo.openstreetmap.org/tags/swimming_pool%3Dkids#wiki but the `kids` are mentioned in https://wiki.openstreetmap.org/wiki/Key:swimming_pool#Alternative_usage

---

Next steps: I think this is not clear enough, yet, to add a preset right now. It would be best if someone where to clarify the tags in the wiki first, likely by starting a forum discussion.

**Comment by Vuizur** [comment]
I agree that there isn't enough consensus yet and an OSM forum discussion is probably the best bet. 

The downside of leisure=paddling_pool is that it isn't rendered by Carto or Organic Maps, but it describes clearly what is meant. Of the other choices, swimming_pool=wading also seems to be good, because it can't get confused with "non-swimmer's pools" as easily as the other choices and has the most uses.

**Comment by matkoniecz** [comment]
Lets close this issue here until consensus emerges, as currently it is no really actionable as far as this project (making iD tagging schema) goes.

Resolving this requires no technical experience, even at level of editing config files required for iD presets
